ldes-server-port-publication-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/LdesFragmentController.java

```diff
@@ -16,10 +19,14 @@ public LdesFragmentController(FragmentProvider fragmentProvider) {
         this.fragmentProvider = fragmentProvider;
     }
 
-    @GetMapping(value = "/ldes-fragment")
-    LdesFragment retrieveLdesFragmentsPage(HttpServletResponse response) {
-        response.setHeader("Content-Type", "application/ld+json; charset=utf-8");
-        return fragmentProvider.getFragment();
+    @GetMapping(value = "/ldes-fragment", produces = "application/ld+json")
```

Comment:
In the future, we should allow a client to set a desired content-type in the request header and dispatch from the incoming request.

Reply:
Since Jena has a list of available conversion types I would somehow limit the freedom of the user to these types.